src/sentry/utils/redis.py

```diff
@@ -100,7 +100,9 @@ def execute_command(self, *args, **kwargs):
 
 class _RedisCluster(object):
     def supports(self, config):
-        return config.get("is_redis_cluster", False)
+        # Default RedisCluster works on both a singular Redis instance
+        # as well as a redis-cluster cluster.
+        return config.get("is_redis_cluster", False) or len(config.get("hosts")) == 1
```

Comment:
The error message that gets raised if this returns `False` never really made much sense, but now it's even more confusing. 😕 

https://github.com/getsentry/sentry/blob/1dcd0c3405778a1a58f7527a63e16e510a6c4491/src/sentry/utils/redis.py#L142-L143

Reply:
Yeah, I saw that. Agreed, but 🤷‍♂.

I think we can follow up and improve this when we remove rb, since we can then be more explicit on what's going on since there will only be 1 class involved.